.github/workflows/release.yaml

```diff
@@ -182,37 +183,137 @@ jobs:
         with:
           version: v0.18.0
 
-      - name: Install cosign
-        uses: sigstore/cosign-installer@398d4b0eeef1380460a10c8013a76f728fb906ac # v3.9.1
-        with:
-          cosign-release: "v2.2.2"
-
       - name: Build and push
         id: publish-image
         env:
           IMAGE_VERSION: ${{ github.ref_name }}
           KO_DOCKER_REPO: "ghcr.io/${{ github.repository }}"
         run: |
-          ko build --sbom=spdx --image-refs ./image-digest --bare --platform linux/arm64,linux/arm/v7,linux/amd64,linux/ppc64le -t ${IMAGE_VERSION} \
+          # Create SBOM directory
+          mkdir -p ./sboms
+
+          # Build multi-platform with SBOMs
+          ko build --sbom=spdx --sbom-dir ./sboms --image-refs ./image-digest --bare --platform linux/arm64,linux/arm/v7,linux/amd64,linux/ppc64le -t ${IMAGE_VERSION} \
```

Comment:
When the fist PR gets merged, I will update this one to remove the deprecated platforms.

Reply:
I've added it to the merge queue, so it should be merged into master within ~7 mins assuming flaky e2e tests don't fail.